jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/impl/TaskQueryServiceImpl.java

```diff
@@ -629,6 +629,16 @@ public int getPendingTaskByUserId(String userId) {
     }
 
     @Override
+    public List<TaskSummary> getTasksByGroup(List<String> groupIds) {
+        Map<String, Object> params = new HashMap<String, Object>();
+        params.put("groupIds", groupIds);
+        return (List<TaskSummary>) persistenceContext.queryWithParametersInTransaction(
+                "TasksByPotentialOwnerGroups",
```

Comment:
Uhm.. If you insist on it, I'm happy to change it. 

However, "TasksByPotentialOwnerGroups" describes exactly what the query does -- whereas "TasksByGroups" implies that the query is something else than what it is! For that reason, I'd like to keep the name to what it is, "TasksByPotentialOwnerGroups".

The operation name (`getTasksByGroup`) was decided by @krisv -- if you think it should be called something else, please talk to him.